### DISCUSSION TITLE
How can I conditionally import a file/module based on the build target?

### DISCUSSION THREAD

**Discussion by danieltroger** [parcel-bundler/parcel#5554] [Q&A] [answered]
For example I have the following target:
```
"ie11_build": {
      "engines": {
        "browsers": "ie 11"
      }
  },
 ```
    
 And there I want to include a polyfill for fetch (i.e. `import 'whatwg-fetch';`) but I do not want that for my modern target.
 
 Do I have to write a script which adds and removes that line from my code and then runs `parcel build --target` ?

**Comment by mischnic** [comment, marked as answer]
That's currently not possible (without writing a custom Parcel plugin).

**Reply by danieltroger** [reply]
I hacked some sauce together which uncomments and comments out a line. Downside is I now have a script which does that and runs parcel with different --targets making the process more convoluted. Can I put it on some list with feature requests?

**Reply by mischnic** [reply]
Can you open an issue for this? And also mention this polyfill-import usecase.
